### ISSUE TITLE
Slow Speed Dart

### ISSUE THREAD

**Issue by maxar6** [team2181/FRC2016#17]
Second feature request by the drive team :D They would like to have a slow speed operation on the dart. Currently the dart speeds are mapped on the second joystick (opjoystick) on the R1 and R2 buttons (the top button and trigger) at 85% speed for up travel and 65% speed for down travel. They would like L1 and L2 mapped to slower speed operation like 50% up speed and 30% down speed. One shower though to add to this, what if we used the analog stick to control up and downward movement. This would allow variable speed control. Let us know what you think and have fun!

**Comment by Brekmister** [comment]
I have already talked to multiple people regarding this. Personally I would
agree to having it be variable but the drive team is insistent on using
just two buttons. This is something I can walk through with  someone
remotely if they want to know whats going on.

**Comment by maxar6** [comment]
I totally agree with you. I guess it does not mean we could not have both so if they prefer one over the other they could use it. If you want to know the current mapping and how it works looks at `OI.cpp` and `ActuateDartCommand.cpp`. Let us know with questions :)
